testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -768,21 +768,22 @@ def __init__(self):
         self.tmpdir = tempdir.TempDir()
         outfile = self.tmpdir.name + '/nucl.pdb'
         u.atoms.write(outfile)
-        self.writtenstuff = open(outfile, 'r').readlines()
+        with open(outfile, 'r') as inf:
+            self.writtenstuff = inf.readlines()
 
     def test_atomname_alignment(self):
         # Our PDBWriter adds some stuff up top, so line 1 happens at [4]
         refs = ("ATOM      1  H5T",
                 "ATOM      2  CA ",
                 "ATOM      3 CA  ",
                 "ATOM      4 H5''",)
-        for written, reference in zip(self.writtenstuff[4:], refs):
+        for written, reference in zip(self.writtenstuff[3:], refs):
```

Comment:
Just to be sure: the first atom is one line earlier than before because it gets written by the single frame version of the PDB writer, and therefore does not have the "MODEL 1" line?